simulation/docker_test.go

```diff
@@ -6,9 +6,14 @@ import (
 
 	"github.com/docker/docker/api/types"
 	"github.com/docker/docker/client"
+	"github.com/ethersphere/swarm/internal/build"
 )
 
 func TestDockerAdapterBuild(t *testing.T) {
+	if env := build.Env(); env.Name == "local" {
```

Comment:
You have a similar skip below this one. Why do we need another one?

Reply:
no its not the same - the skip below is for when you don't have a docker daemon running. i do, and i want to skip the test because i don't want it to constantly pull new images all the time

Reply:
You have a point here, ideally our `unit tests` should not require external dependencies and download files from the internet.

Reply:
ideally we should also consider to have two test suites. i'd be happy if all those simulations are split into a different task that can run separately. i would be happy if i can just run all of unit tests at one run that takes a few seconds to execute rather than sending my machine into a 15 minute meltdown